src/core/diags/README.md

```diff
@@ -77,6 +77,15 @@ Stop transmitting continuous carrier wave.
 Done
 ```
 
+### diag frame \<frame\>
+
+Set the frame(hex encoded) to be used by `diag send` and `diag repeat`.
+
```

Comment:
How to make the diag module to use the previous normal frame format after we set the frame format?